dotcom-rendering/src/web/lib/dynamicSlices.tsx

```diff
@@ -262,37 +231,20 @@ export const Card50_Card25_Card25 = ({
 		<UL direction="row" padBottom={true}>
 			{card50.map((trail) => (
 				<LI percentage="50%" padSides={true}>
-					<FrontCard
+					<Card50Media50
 						trail={trail}
 						containerPalette={containerPalette}
-						containerType={containerType}
 						showAge={showAge}
-						headlineSize="large"
-						imagePosition="top"
-						imagePositionOnMobile="top"
-						supportingContent={trail.supportingContent}
 					/>
 				</LI>
 			))}
 
 			{cards25.map((trail) => (
 				<LI percentage="25%" padSides={true} showDivider={true}>
-					<FrontCard
+					<Card25Media25
```

Comment:
There is some logic about whether to show `trailText` or not being removed in this PR but I cannot see it in [in the wrapper](https://github.com/guardian/dotcom-rendering/blob/b09bf6632ef2bf5e789a03b67750c0f5f3212b84/dotcom-rendering/src/web/lib/cardWrappers.tsx#L182).